system.json

```diff
@@ -21,8 +21,17 @@
         "minimum": 12,
         "verified": "12.331"
     },
-    "esmodules": ["badass.mjs"],
-    "styles": ["css/badass.css"],
+    "esmodules": [
+        "badass.mjs",
+        "scripts/gunCard.js",
```

Comment:
gunCard.js can be removed if the registerSheet is moved to badass.mjs

Reply:
removed/Moved to badass.mjs